content/guest-post/configure-mailazy-as-external-smtp-email-provider/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to Configure Mailazy as an External SMTP Email Provider"
+date: "2021-09-03"
+coverImage: "banner.jpg"
+author: "Kunal"
+tags: ["SMTP", "Transactional Email", "Mailazy"]
+description: "This guide helps you configure LoginRadius for sending authentication and other transactional emails using Mailazy."
+---
+
+A necessary part of handling users in an application is sending them emails. For example, when a user signs up on your app, the user should be sent an email to verify their account. LoginRadius handles email for you out of the box, but you can also easily change this up to use Mailazy as your email provider.
+
+LoginRadius allows configuring your choice of SMTP email provider, so you can manage, monitor, and troubleshoot your email communications.
+
+## Prerequisites
+
+- You must have a configured [Mailazy](https://mailazy.com) account.
+
+## Configure Mailazy
+
+**Step 1:** Sign Up for [LoginRadius Developer Account](https://accounts.loginradius.com/auth.aspx?action=register).
+
+**Step 2:** Log into your [Mailazy](https://app.mailazy.com) account or [sign up](https://app.mailazy.com/signup) for an account if you don’t have one. Go to "Access Keys" and create "Access Keys". Skip if you already have generated access keys.
+
+![mailazy-access-keys](mailazy-access-keys.png)
+
+**Step 3:** Enable and configure Mailazy as your email provider.
+
+Navigate to LoginRadius [Dashboard > Click Select & Configure > Configuration](https://dashboard.loginradius.com/configuration).
+
+![lr-dashboard](lr-dashboard.png)
+
+![lr-dashboard-app](lr-dashboard-app.png)
+
+**Step 4:** Expand "Email/SMS Provider Settings" 
```

Comment:
The steps under `Step 4: Expand "Email/SMS Provider Settings"` have to be updated to match the current flow of adding a SMTP provider in LoginRadius Developer account. Kindly update this.